src/widgets/splits/SplitInput.cpp

```diff
@@ -701,6 +700,28 @@ void SplitInput::editTextChanged()
             app->commands->execCommand(text, this->split_->getChannel(), true);
     }
 
+    if (getSettings()->messageOverflow.getValue() == 2)
```

Comment:
While comparing setting's value, you should not assume "magic values" like `2` here, instead make an `enum`, which can look something like https://github.com/Chatterino/chatterino2/blob/51ece94f588a60c578504309f9eeea22a831f0fa/src/util/StreamerMode.hpp?plain=1#L7 and then compare to, e.g. `FooBar::Highlight`